netket/operator/_sumoperators.py

```diff
@@ -85,7 +98,7 @@ def _expect_kernel_batched(
         return sum(result)
 
     def _pack_arguments(self):
-        return [self._coeff * jnp.array(op._pack_arguments()) for op in self._ops]
+        return list(self._coeff)
```

Comment:
I think this is wrong, because that way you are not passing down the `pack_arguments` of the various terms inside the SumOperator.

Reply:
I do it in the init of the operator now.

Reply:
yeah... i understand it works for your use-case here, but SumOperator is more general and it's supposed to support more arbitrary operators that might not have a single array in their `_pack arguments`. 

Like, stuff with more than 1 array.

Can you please revert this change? I think if you implement it like I propose below it should still work (and it will be less brittle)

Reply:
The coefficient and _pack_arguments is now split. Can you still have a look at the init?